transport/src/main/java/io/netty/channel/pool/SimpleChannelPool.java

```diff
@@ -39,7 +40,8 @@
  *
  */
 public class SimpleChannelPool implements ChannelPool {
-    private static final AttributeKey<SimpleChannelPool> POOL_KEY = AttributeKey.newInstance("channelPool");
+    private final AttributeKey<SimpleChannelPool> POOL_KEY = AttributeKey.newInstance("channelPool." +
```

Comment:
Nit: use lowercase as uppercase is usually only used for static stuff

Reply:
Updated the variable name to reflect its current scope